recipes/tandemtwister/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tandemtwister" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Lionward/tandemtwister.git
+  git_tag: v0.1.0
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Comment:
Could you briefly comment why osx build is skipped?

Btw Bioconda doesn't build for Windows, so it shouldn't be necessary to specify skip win here.